Content.IntegrationTests/Tests/Buckle/BuckleTest.cs

```diff
@@ -21,10 +21,30 @@ namespace Content.IntegrationTests.Tests.Buckle
     [TestOf(typeof(StrapComponent))]
     public class BuckleTest : ContentIntegrationTest
     {
+        private const string PROTOTYPES = @"
+- type: entity
+  name: BuckleDummy
+  id: BuckleDummy
+  components:
+  - type: Buckle
+  - type: Transform
+  - type: Hands
+  - type: Body
+    template: HumanoidTemplate
+    preset: HumanPreset
+    centerSlot: torso
+- type: entity
+  name: StrapDummy
+  id: StrapDummy
+  components:
+  - type: Strap
+  - type: Transform
+";
```

Comment:
Transform is already added to all entities by default.

```suggestion
  - type: Strap
";
```